src/App.js

```diff
@@ -1,11 +1,109 @@
-import './App.css';
+import { Routes, Route } from "react-router-dom";
+import CategoryScroller from "./components/CategoryScroller/CategoryScroller";
```

Comment:
In app.js import files from index.js to make code look clean

Reply:
Done